packages/reference/src/entries/WrappedEntryCard/WrappedEntryCard.tsx

```diff
@@ -93,7 +98,9 @@ export function WrappedEntryCard(props: WrappedEntryCardProps) {
 
   return (
     <EntryCard
-      href={props.getEntryUrl ? props.getEntryUrl(props.entry.sys.id) : undefined}
+      href={
+        props.getEntryUrl && props.isClickable ? props.getEntryUrl(props.entry.sys.id) : undefined
```

Comment:
I think the same should be done for assets if we want to be consistent on how both cards are handled?

i.e. `packages/rich-text/src/plugins/EmbeddedEntityBlock/FetchingWrappedAssetCard.js` and `packages/reference/src/assets/WrappedAssetCard/WrappedAssetCard.tsx`